frontend/packages/dev-console/integration-tests/features/topology/resources-Groupings-ODC-2922.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Topology 
+    User will be able to show and hide all types of resources on Topology graph and list view
+
+
+Background:
+    Given user logged into the openshift application
```

Comment:
We can remove this step, going forward.. As our scripts executes on localhost

Reply:
Should we remove the Background from all the features as we can directly mention it in the Given statement? WDYT @makambalaji